docs/usage/workload_storage.rst

```diff
@@ -0,0 +1,104 @@
+Workload Storage
+================
+
+Any workload needs different volumes with different storage capacities
+to host its components. These volumes are stored in :term:`LVM LV`.
```

Comment:
Consider not using `LVM LV` but `Logical Volume Manager (LVM) Logical Volumes (LVs)`. The reader may not be familiar with the terms at this point.